signprot/templates/signprot/matrix.html

```diff
@@ -24,14 +24,10 @@
     <a id="dlink"></a>
     <div id="excel_table"></div>
 
-
     <ul class="nav nav-tabs" id="mode_nav">
         <li class="active">
             <a href="#single-crystal-group-tab" data-toggle="tab">Single set of structures</a>
         </li>
-        <li class="">
-            <a href="#two-crystal-groups-tab" data-toggle="tab">Two sets of structures</a>
```

Comment:
Depracating the two sets option?

Reply:
This was actually template code that SP copied from the structure comparison tool. As the interface explorer does not have this option and it was interfering with JS calls, I decided to clean it up.